packages/tables/resources/lang/en/table.php

```diff
@@ -25,6 +25,8 @@
 
         'overview' => 'Showing :first to :last of :total results',
 
+        'all' => 'All',
```

Comment:
Wouldn't it be better to move this translation key to `pagination.fields.records_per_page.options.all`?